turbo.json

```diff
@@ -16,6 +16,7 @@
         "generate": {
             "cache": false
         },
-        "test": {}
+        "test": {},
+        "clean": {}
```

Comment:
Turbo's caching system seem to sometimes not want to delete `node_modules` after being ran once. I created a `node_modules` folder in one of the packages and ran the script again, but turbo cached it all and did not want to delete it. We should disable caching by adding a 
```json
"cache": false
```
field to this job

Reply:
done